schema.sql

```diff
@@ -7,4 +7,15 @@ CREATE TABLE animals (
    escape_attempts INT                                              NOT NULL,
    neutered        BOOLEAN                                          NOT NULL,
    weight_kg       DECIMAL                                          NOT NULL
+);
+
+CREATE TABLE owners (
+   id        INT GENERATED ALWAYS AS IDENTITY NOT NULL PRIMARY KEY,
+   full_name TEXT                                         NOT NULL,
```

Comment:
- [ ] It will be nice to use `varchar` datatype for the full_ name here. `varchar` datatype gives you the right to set a max size whereas a `text` has a fixed max size. Take a look at the difference [here](https://www.navicat.com/en/company/aboutus/blog/1308-choosing-between-varchar-and-text-in-mysql#:~:text=Some%20Differences%20Between%20VARCHAR%20and%20TEXT&text=The%20VAR%20in%20VARCHAR%20means,be%20part%20of%20an%20index.)